src/Classes/Particle/Updater.ts

```diff
@@ -67,7 +84,30 @@ export class Updater {
 
                 const ctx = container.canvas.context;
 
-                const colorLine = container.particles.lineLinkedColor;
+                let colorLine: IRgb | undefined;
+
+                /*
+                 * particles connecting line color:
+                 *
+                 *  random: in blink mode : in every frame refresh the color would change
+                 *          hence resulting blinking of lines
+                 *  mid: in consent mode: sample particles color and get a mid level color
+                 *                        from those two for the connecting line color
+                 */
+
+                if (container.particles.lineLinkedColor === "random") {
+                    colorLine = Utils.getRandomColorRGBA();
+                } else if (container.particles.lineLinkedColor == "mid" && particle.color.rgb && p2.color.rgb) {
+                    const sourceColor = particle.color.rgb;
+                    const destColor = p2.color.rgb;
+
+                    colorLine = {
+                        r: Math.floor(Utils.mixComponents(sourceColor.r, destColor.r, particle.radius, p2.radius)),
+                        g: Math.floor(Utils.mixComponents(sourceColor.g, destColor.g, particle.radius, p2.radius)),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'g' is not sorted alphabetically](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)